coeus-s2sgen-impl/src/main/java/org/kuali/coeus/s2sgen/impl/validate/S2SValidatorService.java

```diff
@@ -42,5 +42,18 @@
      * @return validation result true if valid false otherwise.
      */
     boolean validate(XmlObject formObject, List<AuditError> errors);
+    
+    /**
+     * 
+     * This method receives an XMLObject and validates it against its schema and returns the validation result. It also receives a
+     * list in which upon validation failure, populates it with XPaths of the error nodes
+     * 
+     * @param formObject XML document as {@link}XMLObject
+     * @param errors List list of XPaths of the error nodes.
+     * @param formName is the FormName.
+     * @return validation result true if valid false otherwise.
+     */
+    boolean validate(XmlObject formObject, List<AuditError> errors, String formName);
```

Comment:
Could you convert validateUserAttachedFormFile to use this new api.  Then you could remove the old validate method that doesn't take a form name.